### PR TITLE
release: prepare for v0.0.5

### PR DESCRIPTION
**Pull request by aslakknutsen**

### CODE REVIEW

File: docs/modules/ROOT/pages/release_notes/v0.0.5.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+== Highlights of v0.0.5 release
+
+The main focus of this release was to move to the new version of the `Operator SDK` and be able to leverage the `Operator Lifecycle Manager` as an installation process. This will allow us to make the project available for easy consumption via the `Operator Hub`.
+
+image::operator_all_the_things.jpg[Operator All the Things]
+
+== All changes in this release
+
+// autogenerated content will be appended below
```
Comment (bartoszmajsak) [inline docs/modules/ROOT/pages/release_notes/v0.0.5.adoc:9]:
```suggestion
// changelog:generate content will be appended below
```

Comment (bartoszmajsak) [reply docs/modules/ROOT/pages/release_notes/v0.0.5.adoc:9]: To align with coming change (don't forget `/skip-e2e`)

**Comment by alien-ike** [comment]
#### ⚠ Failed generating changelog

#### Found unlabeled PRs

* https://github.com/maistra/istio-workspace/pull/661

**Review by bartoszmajsak — APPROVED** [review]
Agree.gif

**Comment by alien-ike** [comment]
🎇 🎉 🎆 New version has been shipped! 🚀🚀🚀

**Pull request merged**